resources/metadata/integrationArtifactResource.yaml

```diff
@@ -0,0 +1,49 @@
+metadata:
+  name: integrationArtifactResource
+  description: Add, Delete or Update an resource file of integration flow designtime artifact
+  longDescription: |
+    With this step you can either add, delete or update a resource of integration flow designtime artifact using the OData API. Learn more about the SAP Cloud Integration remote API for managing an resource of integration flow artifact [here](https://help.sap.com/viewer/368c481cd6954bdfa5d0435479fd4eaf/Cloud/en-US/d1679a80543f46509a7329243b595bdb.html).
+
+spec:
+  inputs:
+    secrets:
+      - name: cpiApiServiceKeyCredentialsId
+        description: Jenkins secret text credential ID containing the service key to the Process Integration Runtime service instance of plan 'api'
+        type: jenkins
+    params:
+      - name: apiServiceKey
+        type: string
+        description: Service key JSON string to access the Process Integration Runtime service instance of plan 'api'
+        scope:
+          - PARAMETERS
+        mandatory: true
+        secret: true
+        resourceRef:
+          - name: cpiApiServiceKeyCredentialsId
+            type: secret
+            param: apiServiceKey
+      - name: integrationFlowId
+        type: string
+        description: Specifies the ID of the Integration Flow artifact
+        scope:
+          - PARAMETERS
+          - GENERAL
+          - STAGES
+          - STEPS
+        mandatory: true
+      - name: operation
+        type: string
+        description: Specifies the operation for managing resource of the Integration Flow artifact
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+        mandatory: true
+      - name: resourcePath
+        type: string
+        description: Specifies integration artifact resource file relative path.
```

Comment:
Please be more specific in the description. What resource? Maybe an example would be helpful? Are there possible values?

Reply:
this is a file which is uploaded/updated in  integration flow artefact resources, file could be jar, xslt, groovy, java script  files etc

Reply:
If you could give this examples in the description here.